seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/config/StarRocksSinkOptions.java

```diff
@@ -131,10 +129,15 @@ public interface StarRocksSinkOptions {
                     .enumType(StreamLoadFormat.class)
                     .defaultValue(StreamLoadFormat.JSON)
                     .withDescription("");
-
-    SingleChoiceOption<DataSaveMode> SAVE_MODE =
-            Options.key(SupportSaveMode.DATA_SAVE_MODE_KEY)
-                    .singleChoice(DataSaveMode.class, Arrays.asList(DataSaveMode.APPEND_DATA))
+    Option<SchemaSaveMode> SCHEMA_SAVE_MODE =
+            Options.key("schema_save_mode")
+                    .enumType(SchemaSaveMode.class)
+                    .defaultValue(SchemaSaveMode.CREATE_SCHEMA_WHEN_NOT_EXIST)
+                    .withDescription("schema_save_mode");
```

Comment:
I hope there is a more detailed description

Reply:
> I hope there is a more detailed description

Yes, only copy the option name to here doesn't make any sense.

Reply:
updated